src/reconstruction/CoreModelCoupled.jl

```diff
@@ -424,3 +424,36 @@ end
 @_remove_fn metabolite CoreModelCoupled String plural begin
     remove_metabolites(model, Int.(indexin(metabolite_ids, metabolites(model))))
 end
+
+"""
+    change_objective!(
+        model::CoreModelCoupled,
+        rxn_idxs::Vector{Int};
+        weights = ones(length(rxns)),
+    )
+
+Change the objective for `model` to reaction(s) with indices `rxn_ids`, optionally
+specifying their `weights`. By default, assume equal weights. If no objective exists in
+model, sets objective.
+"""
+function change_objective!(
+    model::CoreModelCoupled,
+    rxn_idxs::Vector{Int};
+    weights = ones(length(rxns)),
+)
```

Comment:
forward parameters with `...` and refer to the implementing function (shorter, allows extensibility)